src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
+    Green,

```

Comment:
```suggestion
    GREEN,

```
для єнамі капс неймінг використовується